packages/template/src/project.meta

```diff
@@ -13,15 +13,15 @@
     "audioOffset": 0
   },
   "preview": {
-    "fps": 60,
+    "fps": 30,
```

Comment:
Is this change intended?

Reply:
yeah, this makes the preview in the editor smoother